app/lib/features/attachments/actions/select_attachment.dart

```diff
@@ -1,26 +1,19 @@
-import 'package:acter/common/utils/utils.dart';
 import 'package:acter/features/attachments/types.dart';
 import 'package:acter/features/attachments/widgets/attachment_selection_options.dart';
 import 'package:flutter/material.dart';
 
 Future<void> selectAttachment({
   required BuildContext context,
   required OnAttachmentSelected onSelected,
+  OnLinkSelected? onLinkSelected,
 }) async {
-  context.isLargeScreen
-      ? await showAdaptiveDialog(
-          context: context,
-          builder: (context) => Dialog(
-            child: AttachmentSelectionOptions(
-              onSelected: onSelected,
-            ),
-          ),
-        )
-      : await showModalBottomSheet(
-          isDismissible: true,
-          context: context,
-          builder: (context) => AttachmentSelectionOptions(
-            onSelected: onSelected,
-          ),
-        );
+  await showModalBottomSheet(
```

Comment:
we should really come to an agreement here ... this was just added in a PR yesterday ...

Reply:
Yes true, let's decide about this next product meeting.